erdpy/accounts.py

```diff
@@ -59,13 +64,19 @@ def __init__(self, address: Any = None, pem_file: Union[str, None] = None, pem_i
             address_from_key_file, seed = load_from_key_file(key_file, password)
             self.private_key_seed = seed.hex()
             self.address = Address(address_from_key_file)
+        elif ledger:
+            from erdpy.ledger.ledger_functions import do_get_ledger_address
```

Comment:
why you did the import here inside of the constructor? 
I think it is ok to keep the imports and the beginning of the file.

Reply:
I did this because of cyclic imports. ledger needs a transaction to sign it and the transaction needs ledger logic to handle the signing. writing the imports here helps because the ledger part in imported only when signing

Reply:
done. moved the address fetching a layer above